test/electionguard/mocks/election.cpp

```diff
@@ -13,34 +14,21 @@ extern "C" {
 using namespace electionguard;
 using namespace std;
 
-EG_API eg_electionguard_status_t
-eg_test_election_mocks_get_simple_election_from_file(eg_election_manifest_t **out_handle)
```

Comment:
most of the large code block changes below this file are move refactor changes to match the adjustments made to support c++/cli and changing election to manifest